### PR TITLE
Refactor and polish App README

### PR DESCRIPTION
**Pull request by trisberg**
Relates to spring-cloud-task-app-starters/app-starters-release#39

### CODE REVIEW

File: spring-cloud-starter-task-spark-cluster/README.adoc [hunk 1]
```diff
@@ -21,4 +23,22 @@ $$spark.resource-files$$:: $$A comma separated list of files to be included with
 $$spark.rest-url$$:: $$The URL for the Spark REST API to be used (spark://host:port).$$ *($$String$$, default: `$$spark://localhost:6066$$`)*
 //end::configuration-properties[]
 
+== Example
```
Comment (ilayaperumalg) [inline spring-cloud-starter-task-spark-cluster/README.adoc:26]: @trisberg I can add the `Build with Maven` content here when merging.

Comment (trisberg) [reply spring-cloud-starter-task-spark-cluster/README.adoc:26]: 👍

File: spring-cloud-starter-task-spark-cluster/README.adoc [hunk 2]
```diff
@@ -1,7 +1,9 @@
 //tag::ref-doc[]
 = Spark Cluster Task
 
-This task is intended to launch a Spark application. The task submits the Spark application for execution in a Spark cluster. This task is appropriate for a deployments where any file references can be resolved to a shared location.
+This task is intended to launch a Spark application. 
+The task submits the Spark application for execution in a Spark cluster. 
+This task is appropriate for a deployments where any file references can be resolved to a shared location.
```
Comment (ilayaperumalg) [inline spring-cloud-starter-task-spark-cluster/README.adoc:6]: for a deployments -> for deployments

will fix it while merging.

**Comment by ilayaperumalg** [comment]
Merged as 78c41b7948eea209a98c480edf29ff81ee5f04e0

**Pull request merged**